airbyte-integrations/connectors/destination-pinecone/destination_pinecone/indexer.py

```diff
@@ -25,14 +26,17 @@
 
 MAX_IDS_PER_DELETE = 1000
 
+AIRBYTE_TAG = "airbyte"
+AIRBYTE_TEST_TAG = "airbyte_test"
```

Comment:
I thought it would be good to pass a tag in either case, but we can also not pass a tag during test. lmk what you think.

Reply:
This is great! It's always good to make sure test invocations don't count towards (or against) user activity metrics.